pandas/io/parquet.py

```diff
@@ -252,7 +257,8 @@ def to_parquet(df, path, engine='auto', compression='snappy', index=None,
     ----------
     df : DataFrame
     path : string
-        File path
```

Comment:
I am not sure this doc-string is formatted correctly. @datapythonista ?

also pls list the ``partition_cols`` arg with a versionadded tag (and provide a description).

this only applies to pyarrow? or also fastparquet?

Reply:
this applies only for pyarrow